### PR TITLE
single video page

### PR DESCRIPTION
**Pull request by Chiragmodi01**
- Single video page
- Like page
- Like and dislike feature

### CODE REVIEW

File: src/helpers/services/removeLikeVideoService.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    const encodedToken = localStorage.getItem("token");
+
+    const Headers = { authorization: encodedToken}
+
+    try {
+        const res = await axios.delete(`/api/user/likes/${video._id}`, {headers: Headers});
+        dispatch({ type: "SET_LIKED_ARRAY", payload: res.data.likes})
+    } catch(e) {
+        console.log(e.message)
+    }
```
Comment (harshitpaliwal95) [inline src/helpers/services/removeLikeVideoService.js:13]: use toast in error handling / user not sign in

**Review by harshitpaliwal95 — COMMENTED** [review]
over all code looks good nice ui

File: src/comps/Video/Video.jsx [hunk 2]
```diff
@@ -4,17 +4,20 @@ import './Video.css';
...
-function Video({video, thumbnail, title, channel, timesAgo, timeLength, views}) {
-    const {hideMenu, dispatch, userLoggedIn, incognito} = useMain();
+function Video({video, title, channel, timesAgo, timeLength, views}) {
+    const {hideMenu, dispatch, userLoggedIn, incognito, setHideMenu} = useMain();
     const [showOptions, setShowOptions] = useState(false);
 
     let {pathname} = useLocation();
+    let navigate = useNavigate();
 
     const videoHandler = () => {
```
Comment (aurobindoGupta) [inline src/comps/Video/Video.jsx:17]: finally es6

File: src/comps/Video/Video.jsx [hunk 3]
```diff
@@ -4,17 +4,20 @@ import './Video.css';
 import {BiDotsVerticalRounded} from '../../utils/getIcons';
 import {VideoOptions} from '../../comps/index';
 import { historyVideoService } from '../../helpers/services/historyVideoService';
-import { useLocation } from 'react-router-dom';
+import { useLocation, useNavigate } from 'react-router-dom';
 
                
-function Video({video, thumbnail, title, channel, timesAgo, timeLength, views}) {
-    const {hideMenu, dispatch, userLoggedIn, incognito} = useMain();
+function Video({video, title, channel, timesAgo, timeLength, views}) {
```
Comment (aurobindoGupta) [inline src/comps/Video/Video.jsx]: please es6

**Review by aurobindoGupta — APPROVED** [review]
Keep up the good work.

**Pull request merged**